.github/workflows/scheduled.yml

```
@@ -67,35 +62,50 @@ jobs:
 
           /usr/bin/rake api_samples
 
-          git reset origin/master
-          git add _data/formula{,_canonical.json} _data/bottle api/formula api/bottle formula
-          if ! git diff --no-patch --exit-code HEAD -- _data/analytics _data/formula{,_canonical.json} _data/bottle api/formula api/bottle formula cask; then
-            git commit -m "formula: updates from homebrew/core" _data/analytics _data/formula{,_canonical.json} _data/bottle api/formula api/bottle formula cask
-          fi
+      - uses: actions/upload-artifact@v3
+        if: matrix.os == 'macos-latest'
+        with:
+          name: data-macos
+          path: _data/
+          retention-days: 5
```

Can this retention days be lower?

---

Yeah sure, what did you have in mind?

---

@SMillerDev I guess `1`?
```suggestion
          retention-days: 1
```